### DISCUSSION TITLE
Correction needed in article - GitHub Actions/Creating actions/Create a Docker container action

### DISCUSSION THREAD

**Discussion by theonlyfaddy** [community/community#43209] [GitHub Education]
Team,

There is an error in 

URL : https://docs.github.com/en/actions/creating-actions/creating-a-docker-container-action#example-using-a-private-action

The code snippet highlighted below is not referring to the right tag created and there is a typo by missing out '@'. The example repo being referred to in the article is "hello-world-docker-action" and tag that was created was "v1"

on: [push]

jobs:
  hello_world_job:
    runs-on: ubuntu-latest
    name: A job to say hello
    steps:
      - name: Hello world action step
        id: hello
        uses: **actions/hello-world-docker-actionv2**
        with:
          who-to-greet: 'Mona the Octocat'
      # Use the output from the `hello` step
      - name: Get the output time
        run: echo "The time was ${{ steps.hello.outputs.time }}"

**Comment by airtower-luna** [comment]
Best post that bug as an issue in the https://github.com/github/docs repository, so the team in charge of documentation sees it. :wink:
